openid-federation-1_0.xml

```diff
@@ -5730,11 +5730,10 @@ Content-Type: application/json
       <section title="Key Rollover for a Trust Anchor"
                anchor="key_rollover_anchor">
         <t>
-          A Trust Anchor MUST publish an Entity Configuration about
-          itself. The Trust Anchor SHOULD set an expiration
-          time on its Entity Configuration,
-	  such that federation participants will re-fetch it at reasonable intervals.
-	  When a Trust Anchor rolls over its signing keys, it needs to:
+          A Trust Anchor MUST publish an Entity Configuration about itself. The expiration
+          time (exp) set on this Entity Configuration SHOULD ensure that federation participants
+          re-fetch it at reasonable intervals, as determined by the federation's operational
+          requirements. When a Trust Anchor rolls over its signing keys, it needs to:
```

Comment:
```suggestion
          re-fetch it at reasonable intervals. When a Trust Anchor rolls over its signing keys, it needs to:
```

The Trust Anchor represents the federation and defines the operational requirements, we can say that saying Trust anchor alone is good enough, while including `federation's operational requirements` might introduce some ambiguity or some overlapping elements.